frontend/src/components/UserDetail/EmployeeDetail.tsx

```diff
@@ -180,6 +187,15 @@ const EmployeeDetail = () => {
             alt="availability"
             text={avail === '' ? 'N/A' : avail}
           />
+          {employee.startDate ? (
+            <UserContactInfo
```

Comment:
you can do `employee.startDate && <UserContactInfo ... />` instead of rendering an empty div